src/data-and-backend/firebase.md

```diff
@@ -3,28 +3,49 @@ title: Firebase
 description: How to use Firebase and Firestore with Flutter.
 ---
 
+{% include docs/yt_shims.liquid %}
+
+== Introduction
```

Comment:
Were you intending for these to be headers or some other new format?

If headers, please update to hash symbols for them to work as Markdown headers.

Reply:
Yes, this is the first I've seen of new headers. We need to discuss this.

Reply:
I mixed up MediaWiki format with Markdown. I'll fix it.